phpunit.xml.dist

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit colors="true" bootstrap="vendor/autoload.php">
+    <php>
+        <!-- Disable E_USER_DEPRECATED until reaching compatibility with Symfony 3.0 -->
+        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
+        <ini name="error_reporting" value="-16385"/>
```

Comment:
IMO, we should not lock <2.7 in the composer.json, otherwise people wanting to try Symfony 2.7 already (ignoring the deprecation warnings for now) would be unable to use the latest version of the bundle (they will be forced to use the 1.3.0 release, and they will then complain about bugs already fixed in newer versions of the bundle once we merge some bug fixes).
Otherwise, we should exclude 2.7 only in Travis (or in dev requirements)